src/components/Registration/RegistrationContainer.tsx

```diff
@@ -144,6 +144,29 @@ export const RegistrationContainer = withRouter(props => {
     }
   })(props.match.params.signUpStep);
 
+  const rightColumnContent = (step => {
+    switch (step) {
+      case "1":
+        return (
+          <Fragment>
+            <p className=" pr-3 pt-4">Guida all'utilizzo</p>
+            <p className="small pr-3">
+              Le informazioni legate al tuo ente vengono derivate dal database
```

Comment:
Should the text be in the HTML?

Reply:
there are no html files in a React project, you create everythin in javascript/jsx (or in this case typescript/tsx) files